### PR TITLE
People: Invites: Display notices after redirect

### PR DESCRIPTION
**Pull request by lezama**
__How to test__

- Checkout `update/invite-notices` branch
- Go to `$site/wp-admin/users.php?page=wpcom-invite-users`
- Invite a user
- Log in as the user
- In the invite email, make note of the invitation key in the activation link
- Go to `/accept-invite/$site/$invite_key`
- Accept the invite
- Assert that the notice appears only after being redirected to the site posts list

### CODE REVIEW

**Comment by lezama** [comment]
cc @ebinnion @roccotripaldi

File: client/boot/index.js [hunk 1]
```diff
@@ -313,6 +310,10 @@ function boot() {
 
 	require( 'my-sites' )();
 
+	// clear notices
+	//TODO: remove this one when notices are reduxified - it is for old notices
+	page( '*', require( 'notices' ).clearNoticesOnNavigation );
+
 	if ( config.isEnabled( 'olark' ) ) {
```
Comment (roccotripaldi) [inline client/boot/index.js]: Can you explain why we are moving this?

**Comment by roccotripaldi** [comment]
Code looks good, and it works well. You have my :+1:

File: client/boot/index.js (hunk 1 above)
Comment (lezama) [inline client/boot/index.js]: In order to make `displayOnNextPage` property to work as expected. If we don't do this we clear the notices we render the scheduled ones before rendering "mysites"

Comment (roccotripaldi) [inline client/boot/index.js]: Makes sense! Thanks.

**Comment by lezama** [comment]
Fixes #1236

**Pull request merged**